fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -78,5 +84,19 @@ def display_user_error!(e, message)
     def reraise_formatted!(e, message)
       raise e, "[!] #{message}".red, e.backtrace
     end
+
+    def show_github_issues(message)
+      return if ENV["FASTLANE_HIDE_GITHUB_ISSUES"]
+      return if FastlaneCore::Helper.test?
+
+      require 'gh_inspector'
+      require 'fastlane_core/ui/github_issue_inspector_reporter'
+
+      inspector = GhInspector::Inspector.new("fastlane", "fastlane", verbose: $verbose)
+      delegate = Fastlane::InspectorReporter.new
+      inspector.search_query(message, delegate)
```

Comment:
if you have an error object, might be better to use `search_exception` ( perhaps that's named badly ) as it can then remove instance var addresses and can do some other useful work in the future too